Rlabkey/man/labkey.provenance.createProvenanceParams.Rd

```diff
@@ -0,0 +1,64 @@
+\name{labkey.provenance.createProvenanceParams}
+\alias{labkey.provenance.createProvenanceParams}
+\title{Create provenance parameter object}
+\description{
+Helper function to create the data structure that can be used in provenance related APIs.
+Note: this function is in beta and not yet final, changes should be expected so exercise caution when using it.
+}
+\usage{
+labkey.provenance.createProvenanceParams(recordingId=NULL, name=NULL, description=NULL,
+        materialInputs=NULL, materialOutputs=NULL, dataInputs=NULL, dataOutputs=NULL,
+        inputObjectUriProperty=NULL, outputObjectUriProperty=NULL, objectInputs=NULL, objectOutputs=NULL,
+        provenanceMap=NULL)
+}
+\arguments{
+  \item{recordingId}{(optional) the recording ID to associate with other steps using the same ID}
+  \item{name}{(optional) the name of this provenance step}
+  \item{description}{(optional) the description of this provenance step}
+  \item{materialInputs}{(optional) the list of materials (samples) to be used as the provenance run input. The data structure should be a dataframe with the column name describing the data type (lsid, id)}
```

Comment:
Maybe you already ran the CRAN check but I assume they are going to get mad about the number of characters on various lines in these files. We can fix those up later though.